app/services/ratings.js

```diff
@@ -0,0 +1,47 @@
+const { User, Weet, Rating } = require('../models');
+const { sequelize } = require('../models');
+const logger = require('../logger');
+
+exports.findRateWeet = async (id, ratingUserId) => {
+  const rating = await Rating.findOne({ where: { weetId: id, ratingUserId } });
+  return rating;
+};
+
+exports.createRateWeet = async (previousRateId, weetId, body) => {
+  let transaction = {};
+  const { score, ratingUserId } = body;
+  try {
+    transaction = await sequelize.transaction();
```

Comment:
You are not using the transaction, are you?

Reply:
I thought it worked 😧 , now I see the error.

Reply:
I didn't pass the transaction in the options.